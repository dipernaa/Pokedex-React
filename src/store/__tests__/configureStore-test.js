import configureStore from '../configureStore';

it('should setup middleware and call onComplete', (done) => {
  const store = configureStore(() => {
    done();
  });

  expect(store).toMatchSnapshot();
});
