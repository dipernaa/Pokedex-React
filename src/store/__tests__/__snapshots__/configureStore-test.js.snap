// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should setup middleware and call onComplete 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
